frontend/src/metabase/setup/components/Setup.jsx

```diff
@@ -24,6 +26,15 @@ const DATABASE_CONNECTION_STEP_NUMBER = 3;
 const DATABASE_SCHEDULING_STEP_NUMBER = 4;
 const PREFERENCES_STEP_NUMBER = 5;
 
+const AddDatabaseHelpCardHolder = styled.div`
```

Comment:
Is it worth adopting the `setup.styled.js` convention here and moving this to its own file?

Reply:
Oh yeah, thanks for keeping an eye on that. I'll create a separate `styled` file for the card component itself